pytest/tests/loadtest/locust/README.md

```diff
@@ -0,0 +1,36 @@
+# Locust based load testing
+
+WIP: This is work in progress, motivation described in https://github.com/near/nearcore/issues/8999
+
+TLDR: Use [locust](https://locust.io/) to generate transactions against a Near chain and produce statistics.
+
+## Install
+```
+pip3 install locust
+```
+
+## Run a first load test
+
+The load generator needs access to a FT contract WASM and it needs access to a account key with plenty of tokens.
+For a local test setup, this works just fine.
+```
+CONTRACT=runtime/near-test-contracts/res/fungible_token.wasm
+KEY=~/.near/test0/validator_key.json
+```
+
+For a quick demo, you can also run a local cluster using the provided script.
+```
+NEAR_ROOT=./target/release/  python3 ./pytest/tools/locust/local_cluster.py --shards 4
+```
+
+Then to actually run it, this is the command.
+```
```

Comment:
Specifying that this is a bash code block by starting it with " ```sh " would yield better code highlighting:

```sh
cd pytest/tests/loadtest/locust/
locust -H 127.0.0.1:3040 \
  --fungible-token-wasm=$CONTRACT \
  --contract-key=$KEY
```